horace_core/sqw/@sqw/tobyfit_DGfermi_resfun_covariance.m

```diff
@@ -110,6 +110,7 @@
 
 
     cov_hkle{iw} = transform_matrix (cov_x, dq_mat);
+    %TODO: Re #1040 this code is not consistent with generic projection:
```

Comment:
so if it isn't consistent, the issue says they should be made consistent. In the meantime, are we allowing incorrect calculation, or a crash? Shouldn't we have an error if non-orthogonal?

Reply:
fixed in #892